packages/runtime/src/prop-controllers/introspection.ts

```diff
@@ -212,6 +217,57 @@ export function getElementSwatchIds<T extends Data>(
       return value?.swatchId == null ? [] : [value.swatchId]
     }
 
+    case StyleV2ControlType: {
```

Comment:
These are not the permanent locations. I am moving onto those introspection tickets at which time I'll move all these cases out/ colocate with them with their domain